config/static_caching.php

```diff
@@ -12,7 +12,7 @@
     |
     */
 
-    'strategy' => null,
+    'strategy' => env('STATAMIC_CACHING_STRATEGY', true),
```

Comment:
If you change it to `STATAMIC_STATIC_CACHING_STRATEGY`, and fall back to `null`, we're in business.

Reply:
I just saw that and did already change it. Sorry.

Reply:
You're still missing the `STATIC` in the key

Reply:
Closes to midnight over here. Too late as you can see 😅

Thanks and good night 😴